shared/middleware/helpers/core.rb

```diff
@@ -46,9 +46,19 @@ def not_found()
 end
 
 def too_large()
-  halt(413, "Payload too large\n")
```

Comment:
hm, is this used other places that is expecting the halt behavior? My concern is that changing this method makes it different from the other nearby methods.